pool/src/pool.rs

```diff
@@ -409,6 +409,74 @@ where
 		}
 	}
 
+	/// Attempt to deaggregate a transaction and add it to the mempool
+	pub fn deaggregate_and_add_to_memory_pool(
+		&mut self,
+		tx_source: TxSource,
+		tx: transaction::Transaction,
+		stem: bool,
+	) -> Result<(), PoolError> {
+		match self.deaggregate_transaction(tx.clone()) {
+			Ok(deaggragated_tx) => self.add_to_memory_pool(tx_source, deaggragated_tx, stem),
+			Err(e) => {
+				debug!(
```

Comment:
maybe make this a warning! level log?

Reply:
Yeah, we can expect this to happen quite often once we have the Dandelion transaction aggregation, so ```DEBUG``` is fine for now.